contracts/sol6/KyberStorage.sol

```diff
@@ -585,6 +633,24 @@ contract KyberStorage is IKyberStorage, PermissionGroupsNoModifiers, Utils5 {
             // if reserve wasn't found add it
             reserveArr.push(reserveId);
         }
+
+        for (i = 0; i < tokensArr.length; i++) {
+            if (token == tokensArr[i]) {
+                if (add) {
+                    break; // already added
```

Comment:
regarding this can previous same code.

if already added. can we exit this function?

so this question actually holds for line 622

if we can current loop is enough for remove only option.

Reply:
Mmm yes, it makes sense